src/Compilers/CSharp/Test/Semantic/Semantics/AccessCheckTests.cs

```diff
@@ -467,16 +456,7 @@ public void X()
         [Fact]
         public void AccessCheckProtectedColorColor()
         {
-            // SPEC VIOLATION: The specification implies that first overload resolution chooses the
-            // SPEC VIOLATION: best method, and then if the receiver is the wrong type and the
-            // SPEC VIOLATION: best method is protected, then an error occurs. The native compiler
-            // SPEC VIOLATION: does it in the other order: first protected methods with potentially
-            // SPEC VIOLATION: the wrong receiver type are discarded from the candidate set, and
-            // SPEC VIOLATION: then the best method is chosen.
-            // SPEC VIOLATION: We should consider changing the specification to match the native
-            // SPEC VIOLATION: compiler behavior; it is arguably sensible and would be a 
-            // SPEC VIOLATION: bad breaking change to fix now.
-            //
+            // Inaccessible methods are not a member of a method group.
```

Comment:
>// [](start = 12, length = 2)

There's no longer a spec violation here?

Reply:
There never was. The comment was written based on a misunderstanding of the spec. A method group only contains accessible methods, and that is what the native compiler and Roslyn implement by rejecting protected methods with the wrong receiver type.

---
In reply to: [202478718](https://github.com/dotnet/roslyn/pull/27312#discussion_r202478718) [](ancestors = 202478718)